### PR TITLE
fixes for sanitize_and_translate

### PR DESCRIPTION
**Pull request by SuperFirm84**

### CODE REVIEW

File: app/common/translate.py [hunk 1]
```diff
@@ -548,8 +615,7 @@ def clean_up_and_return_items(text: str) -> str:
     sanitized = re.sub("男は ", "", text)  # remove boy reference from start of string
     sanitized = re.sub("女は ", "", sanitized)  # remove girl reference from start of string
     sanitized = re.sub("男は　", "", sanitized)  # remove boy reference from start of string (fullwidth space)
-    sanitized = re.sub("女は　", "", sanitized)  # remove girl reference from start of string (fullwidth space)
-    sanitized = re.sub("！　と頼まれた。", "と頼まれた。", sanitized)  # remove annoying 'asked to' string
```
Comment (jmctune) [inline app/common/translate.py:552]: What did this do to change the behavior of the text? This was annoying to see "I was asked to do this" a bunch.

File: app/common/translate.py [hunk 2]
```diff
@@ -548,8 +615,7 @@ def clean_up_and_return_items(text: str) -> str:
...
+    sanitized = re.sub("女は　", "", sanitized)  # remove girl reference from s tart of string (fullwidth space)
```
Comment (jmctune) [inline app/common/translate.py:618]: Can you fix this comment? `s tart` -> `start

File: app/common/translate.py [hunk 3]
```diff
@@ -220,13 +274,18 @@ def sanitize_and_translate(self, text: str, wrap_width: int, max_lines=None, add
         for alignment in alignments:
             output = output.replace(alignment, "")
 
+        # trim multiple ellipses to a single one
+        ellipses = ["…………………………………………", "………………………………………", "……………………………………", "…………………………………", "………………………………", "……………………………", "…………………………", "………………………", "……………………", "…………………", "………………", "……………", "…………", "………", "……"]
```
Comment (jmctune) [inline app/common/translate.py:278]: This is a long list -- can you format this more like:

```python
ellipses = [
  "…………………………………………",
  "………………………………………",
  ..
  ..
]
```

File: app/common/translate.py (hunk 1 above)
Comment (SuperFirm84) [reply app/common/translate.py:552]: This never worked here anyways but also we do this in the glossary now.
と頼まれた。,""

File: app/common/translate.py (hunk 2 above)
Comment (SuperFirm84) [reply app/common/translate.py:618]: Yep

File: app/common/translate.py (hunk 3 above)
Comment (SuperFirm84) [reply app/common/translate.py:278]: Yep

File: app/common/translate.py [hunk 4]
```diff
@@ -220,13 +274,34 @@ def sanitize_and_translate(self, text: str, wrap_width: int, max_lines=None, add
         for alignment in alignments:
             output = output.replace(alignment, "")
 
+        # trim multiple ellipses to a single one
+        ellipses = [
```
Comment (jmctune) [inline app/common/translate.py:278]: Each of these lines needs to be indented by 4 spaces.

Comment (SuperFirm84) [reply app/common/translate.py:278]: Right, mb

**Review by jmctune — APPROVED** [review]

**Pull request merged**